holidays/countries/vietnam.py

```diff
@@ -32,47 +64,81 @@ class Vietnam(ObservedHolidayBase, ChineseCalendarHolidays, InternationalHoliday
     def __init__(self, *args, **kwargs):
         ChineseCalendarHolidays.__init__(self)
         InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, VietnamStaticHolidays)
         kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_WORKDAY)
+        kwargs.setdefault("observed_since", 1995)
         super().__init__(*args, **kwargs)
 
+    def _add_lunar_new_year_observed(self, dt_lny: date) -> None:
+        if self._year <= 1994:
+            return None
+
+        day_names = {
+            # 29 of Lunar New Year.
+            -2: tr("29 Tết"),
```

Comment:
This name (and its translation) are based on [this comment](https://github.com/vacanza/holidays/pull/2025#issuecomment-2387386364). 

To be honest, I don't quite understand the logic behind this name (why the second-to-last day of the year is called the 29th of something), so I'm asking @vunhatchuong to explain in more detail if possible.

Reply:
I name it this way because there are no official names for it and this is a common way websites also refer to it.

The 29th could either be the second-to-last day of the year (2024) or the last day of the year (2022) because of how the Lunar calendar works so you also can't simply call it "Lunar New Year's Eve".

When you talk about Tết, people only keep track of at most the first 5 days so there are no misunderstandings of the name "29 Tết".

Reply:
@vunhatchuong , thank you for your answer! So let's use this name.